edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudConnectionProvider.cs

```diff
@@ -46,7 +49,9 @@ public class CloudConnectionProvider : ICloudConnectionProvider
             TimeSpan operationTimeout,
             bool useServerHeartbeat,
             Option<IWebProxy> proxy,
-            IMetadataStore metadataStore)
+            IMetadataStore metadataStore,
+            bool scopeAuthenticationOnly = true,
+            bool trackDeviceState = false)
```

Comment:
Do these have default values because of tests?

Reply:
yes. I changed to use explicit values